pkg/webhook/resources/setting/validator.go

```diff
@@ -57,7 +57,12 @@ const (
 	labelAppNameValueAlertManager     = "alertmanager"
 	labelAppNameValueGrafana          = "grafana"
 	labelAppNameValueImportController = "harvester-vm-import-controller"
-	maxTTLDurationMinutes             = 52560000 //specifies max duration allowed for kubeconfig TTL setting, and corresponds to 100 years
+
+	vClusterNamespace          = "rancher-vcluster"
```

Comment:
please check & move / reuse those labels to/from harvester const for common usage, thanks.